Tables.lua

```diff
@@ -3,11 +3,13 @@ local _, e = ...
 local function ParseOnlineUnits(A)
 	local tbl = {}
 	local units = {}
-	local name
+	local name, online
 	if e.FrameListShown() == 'guild' then
-		for i = 1, select(2, GetNumGuildMembers()) do
```

Comment:
That only returns the number of online guild members, which ends up restricting the amount of people we loop over and query w/ GetGuildRosterInfo(), so we end up an incomplete dataset.